gapic/api/artman_logging.yaml

```diff
@@ -9,8 +9,10 @@ common:
   output_dir: ${REPOROOT}/artman/output
   gapic_api_yaml:
     - ${REPOROOT}/googleapis/google/logging/v2/logging_gapic.yaml
+  enable_batch_generation: True
 java:
   final_repo_dir: ${REPOROOT}/gcloud-java/gcloud-java-logging
+  batch_dir_name: gcloud-java/gcloud-java-logging
```

Comment:
It doesn't seem like it's necessary to configure this. I think we have enough information to make a useful path - maybe something like using ${language}/${api_name}.